modules/imaging_uploader/js/imaging_uploader.js

```diff
@@ -215,6 +215,7 @@ function UploadProgress() {
     this.getProgressText = function() {
         var columns    = $(this._uploadRow).find('td'),
             uploadId   = $.trim($(columns).eq(1).text()),
+            candid     = $.trim($(columns).eq(3).text()),
```

Comment:
I can't comment on the line above, but there seems to be a typo on line 167 that I noticed when trying to figure out what these variables are.. "this. _uploadRow" has a space between . and _